router/parser/receiver_parser.go

```diff
@@ -2,64 +2,77 @@ package parser
 
 import (
 	"fmt"
+	"strconv"
 	"strings"
-
-	sdk "github.com/cosmos/cosmos-sdk/types"
+	"time"
 )
 
 type ParsedReceiver struct {
 	ShouldForward bool
 
-	HostAccAddr sdk.AccAddress
-	Destination string
-	Port        string
-	Channel     string
+	HostAccAddr    string
+	Destination    string
+	Port           string
+	Channel        string
+	Timeout        time.Duration
+	ForwardRetries *uint8
 }
 
 // For now this assumes one hop, should be better parsing
```

Comment:
This comment does not apply anymore, right?

Reply:
Right, but no need to worry about the receiver field since we are removing it in favor of the memo field in the next PR